### ISSUE TITLE
Setup failed for zha_new: Could not install all requirements.

### ISSUE THREAD

**Issue by bakazm** [Yoda-x/ha-zha-new#64]
On a fresh install using the Raspberry Pi 3 image, I am getting the following error. I don't think I can run any pip3 commands because the rasp pi image's SSH is limited:

> 2018-11-25 17:59:23 WARNING (MainThread) [homeassistant.loader] You are using a custom component for zha_new which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you do experience issues with Home Assistant.
2018-11-25 18:01:38 ERROR (SyncWorker_6) [homeassistant.util.package] Unable to install package https://github.com/Yoda-x/zigpy/archive/master.zip#zigpy==0.1.4-Y: Failed building wheel for pycryptodome
python-jose-cryptodome 1.3.2 has requirement pycryptodome<3.4.0,>=3.3.1, but you'll have pycryptodome 3.7.1 which is incompatible.
pyrebase4 4.2.0 has requirement oauth2client==4.1.2, but you'll have oauth2client 4.0.0 which is incompatible.
broadlink 0.9 has requirement pycryptodome==3.4.11, but you'll have pycryptodome 3.7.1 which is incompatible.
Command "/usr/local/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-_b_50i3a/pycryptodome/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-l8yueimm/install-record.txt --single-version-externally-managed --prefix  --compile --user --prefix=" failed with error code 1 in /tmp/pip-install-_b_50i3a/pycryptodome/
You are using pip version 18.0, however version 18.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
2018-11-25 18:01:38 ERROR (MainThread) [homeassistant.requirements] Not initializing zha_new because could not install requirement https://github.com/Yoda-x/zigpy/archive/master.zip#zigpy==0.1.4-Y
2018-11-25 18:01:38 ERROR (MainThread) [homeassistant.setup] Setup failed for zha_new: Could not install all requirements.

**Comment by Yoda-x** [comment]
ok, seems you have the newest hass , let me check when the requirement changed and what changed. My requirements seems outdated...

**Comment by Yoda-x** [comment]
I take a look in this.
I could not see the same on my venv system on newest hass. 
What components are enabled in your system or do you use discovery?

**Comment by bakazm** [comment]
This is a completely clean install of the latest Raspberry Pi image using Hass 0.82.1. The only component I have is zha_new and discovery is enabled. The only add-ons that are installed are Configurator and Samba.

**Comment by Yoda-x** [comment]
Can you please disable discovery and see if the error persists...

**Comment by bakazm** [comment]
I disabled discovery (commented out the discovery line in the configuration.yaml) and rebooted. I got the same error.

**Comment by bakazm** [comment]
I used the SSH addon to roll back to 80.3. Rebooted and got the same errors.

**Comment by bakazm** [comment]
I joined the beta channel and upgraded home assistant to 0.83.0b1. The zha_new component finally loaded correctly. I'll add some devices tomorrow to confirm that everything is working. Thanks!

**Comment by bakazm** [comment]
I just confirmed that under the 0.83.0b1 build, zha_new loads correctly and is able to interface with my aqara motion sensors.

**Comment by ryanjkelly** [comment]
I'm having the same problem and am running **Hassio** on **Raspberry Pi 3 B+**. I've tried on both `0.84` and `0.85`.

`0.86` was just released today, so I'll try again on that and report back.

Since I'm using Hassio, my command list is limited, so I can't upgrade or downgrade `pycryptodome` or `pip`.

**Comment by Yoda-x** [comment]
did you tried mainstream zha ? the pycryptodome depdencency is in both version (my and mainstream) possible that hassio is broken again. if you have the same error on the mainstream zha, please open a bug inside homeassistant.

**Comment by ryanjkelly** [comment]
Thanks, I was able to get it working on `0.86.3`, but my SmartThings Multipurpose sensor (CentraLite 3321-S) was reporting "unknown" for its temperature state, and I was having trouble pairing some Xiaomi sensors and switches I recently got from AliExpress.

I think I'm going to try out Koen's [Zigbee2mqtt](https://github.com/Koenkk/zigbee2mqtt) solution and see how that works out. Thanks again for your help.
